conf/modules.config

```
@@ -81,6 +85,16 @@ process {
         ext.prefix = { "${input.baseName}" }
     }
 
+    withName: CRUMBLE {
+        ext.prefix = { "${input.baseName}.crumble" }
+        ext.args   = '-y pbccs -O cram'
+        publishDir = [
+            path: { "${params.outdir}/read_mapping/crumble" },
```

The crumbled file is meant to replace the uncompressed cram file, so it should be in the same directory
```suggestion
            path: { "${params.outdir}/read_mapping/pacbio" },
```

---

For now we are publishing both CRAM and crumble CRAM. Once we add crumble feature for the other technologies and add the output options, I will update this. Right now I don't know what that will look like.

---

The requirement is to have the directory called `read_mapping/${library_type}`. The names of the crumbled files don't clash with the pre-crumble files, so all could co-exist within `pacbio/`

---

Done ✅